lib/alexa.js

```diff
@@ -367,6 +367,9 @@ function ResponseBuilder(self) {
 
                 responseObject.response.directives = [audioPlayerDirective];
                 return this;
+            },
+            'setResponse': function () {
```

Comment:
Odd, I did some testing, and it definitely wasn’t changing this.handler.response. I’m working on some other things right now, but I’ll have a look into this...